pkg/debug/checkups/power_windows.go

```diff
@@ -0,0 +1,64 @@
+//go:build windows
+// +build windows
+
+package checkups
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"syscall"
+
+	"github.com/kolide/launcher/pkg/agent"
+)
+
+type powerCheckup struct{}
+
+func (p *powerCheckup) Name() string {
+	return "Power Report"
+}
+
+func (p *powerCheckup) Run(ctx context.Context, extraWriter io.Writer) error {
+	// Create a temporary file for powercfg to write its output to
+	tmpFilePath := agent.TempPath("launcher-checkup-spr.html")
+	defer func() {
+		os.Remove(tmpFilePath)
+	}()
```

Comment:
```suggestion
	defer os.Remove(tmpFilePath)
```

maybe?

Reply:
Oh yup, no good reason to do it the way I have it now, I'll update